src/ontology/mondo-edit.obo

```diff
@@ -126785,12 +126782,9 @@ xref: UMLS:C0033806 {source="MONDO:ncbi_mim2gene_medline"}
 xref: UMLS:C2931404 {source="MONDO:equivalentTo"}
 xref: UMLS:C3494506 {source="OMIM:103580", source="Orphanet:79443/e", source="MONDO:equivalentTo", source="NCIT:C129721", source="Orphanet:79443"}
 is_a: MONDO:0002081 ! musculoskeletal system disorder
-is_a: MONDO:0016565 {source="MONDO:0018699-obsoleted", source="https://github.com/monarch-initiative/mondo/pull/2571/"} ! syndromic genetic obesity
 is_a: MONDO:0019695 {source="MONDO:0018699-obsoleted", source="https://github.com/monarch-initiative/mondo/pull/2571/"} ! acromelic dysplasia
 is_a: MONDO:0019992 {source="MONDO:0018699-obsoleted", source="https://github.com/monarch-initiative/mondo/pull/2571/"} ! pseudohypoparathyroidism
 is_a: MONDO:0800094 {source="PMID:31633310"} ! dysostosis with brachydactyly with extraskeletal manifestations
-relationship: disease_has_feature HP:0001513 ! Obesity
```

Comment:
I removed this (and the line below) because it was creating an incorrect inferred parent.